crates/sui-types/src/temporary_store.rs

```diff
@@ -557,13 +564,18 @@ impl<S> TemporaryStore<S> {
             .or_else(|| self.input_objects.get(id))
     }
 
-    pub fn apply_object_changes(&mut self, changes: BTreeMap<ObjectID, ObjectChange>) {
+    pub fn apply_object_changes(
+        &mut self,
+        changes: BTreeMap<ObjectID, ObjectChange>,
+        loaded_child_objects: Option<BTreeMap<ObjectID, SequenceNumber>>,
```

Comment:
I feel like this should really just be in a separate API, as it isn't entirely connected